ui/src/app/pages/homepage/data-access-requirements.spec.tsx

```diff
@@ -931,5 +903,95 @@ describe('DataAccessRequirements', () => {
         wrapper = component();
         await waitOneTickAndUpdate(wrapper);
         expect(findModule(findControlledTierCard(wrapper), AccessModule.CTCOMPLIANCETRAINING).exists()).toBeFalsy();
-    });
+      });
+
+  it('Should display CT training when ineligible', async() => {
+    let wrapper = component();
+    await waitOneTickAndUpdate(wrapper);
+
+    profileStore.set({
+      profile: {
+        ...ProfileStubVariables.PROFILE_STUB,
+        tierEligibilities: [{
+          accessTierShortName: AccessTierShortNames.Registered,
+          eraRequired: false,
+          eligible: false
+        },{
+          accessTierShortName: AccessTierShortNames.Controlled,
+          eraRequired: true,
+          eligible: false
+        }]
+      },
+      load,
+      reload,
+      updateCache
+    });
+    wrapper = component();
+    await waitOneTickAndUpdate(wrapper);
+    expect(findIneligibleModule(wrapper, AccessModule.CTCOMPLIANCETRAINING).exists()).toBeTruthy();
+  });
+
+  it('Should display CT training when eligible', async() => {
+    let wrapper = component();
+    await waitOneTickAndUpdate(wrapper);
+
+    profileStore.set({
+      profile: {
+        ...ProfileStubVariables.PROFILE_STUB,
+        tierEligibilities: [{
+          accessTierShortName: AccessTierShortNames.Registered,
+          eraRequired: false,
+          eligible: false
+        },{
+          accessTierShortName: AccessTierShortNames.Controlled,
+          eraRequired: true,
+          eligible: true
+        }]
+      },
+      load,
+      reload,
+      updateCache
+    });
+    wrapper = component();
+    await waitOneTickAndUpdate(wrapper);
+    expect(findIncompleteModule(wrapper, AccessModule.CTCOMPLIANCETRAINING).exists()).toBeTruthy();
+  });
+
+  it('Should allow CT and DUCC as simultaneously active', async() => {
```

Comment:
active -> clickable.  I'd also like a test for "does not allow both to be active"